pkg/plugin/backup_pvc_action_plugin.go

```diff
@@ -100,15 +100,27 @@ func (p *NewPVCBackupItemAction) Execute(item runtime.Unstructured, backup *vele
 		return nil, nil, errors.WithStack(err)
 	}
 
+	// Do nothing if restic is used to backup this PV
```

Comment:
Can you remove #460 from this commit message?

Reply:
done